register/models.py

```diff
@@ -56,11 +56,13 @@
 D_NONE = 'None'
 D_VEGETERIAN = 'Vegeterian'
 D_GLUTEN_FREE = 'Gluten-free'
+D_OTHER = 'Others'
 
 DIETS = [
-    (D_NONE, 'None'),
+    (D_NONE, 'No requirements'),
```

Comment:
This is personal preference because of the wording - even a "regular" diet is a diet, so _None_ sounded a bit weird. But I'm okay with leaving it as it was.